test/functionalTest/cases/1705_csub_cache_objects/csub_cache_objects.test

```diff
@@ -0,0 +1,57 @@
+# Copyright 2016 Telefonica Investigacion y Desarrollo, S.A.U
```

Comment:
The missing test is to create a subscription with a string filter and then update it (with a different string filter) and make sure it works with the new filter.
I have already seen in the source code that it is taken care of, but a test is always good to 'protect it' against future bugs.